common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1613,7 +1613,7 @@ def get_course_template_context(self):
         request.user = user
         user_info = cc.User.from_django_user(self.runtime.user).to_dict()
         course_id = self.runtime.course_id
-        course = get_course_with_access(self.runtime.user, 'load_forum', course_id)
+        course = get_course_with_access(self.runtime.user, 'load', course_id)
```

Comment:
@Kelketek Do you think these also need to specify `check_if_enrolled`? See [here](https://github.com/edx/edx-platform/commit/d240785b17bdcaeb11e4ef24ee4a3697da26c9b4#diff-1f8b9b8344cecfd83d74693377466fd4) for why I ask.